### PR TITLE
Sort timeline itens by date

### PR DESCRIPTION
**Pull request by venkateshwarv**
Sorts timeline events in Timeline widget and customer journey widget.

### CODE REVIEW

File: CustomerJourney/package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "lit-html": "1.2.1"
   },
   "peerDependencies": {
-    "@cjaas/common-components": "4.0.1",
+    "@cjaas/common-components": "4.1.1-dev1",
```
Comment (elenagiere) [inline CustomerJourney/package.json:29]: just remember to update this to a real version, not a dev version once all is approved and merged.

File: Timeline/package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     "dist": "npm run check && webpack --config-name=distProd",
     "dist:dev": "npm run check && webpack --config-name=distDev",
     "dist:watch": "npm run check && webpack --config-name=distDevWatch",
-    "check": "check-peer-dependencies --yarn",
+    "check": " ",
```
Comment (elenagiere) [inline Timeline/package.json:19]: why remove this?

File: Timeline/src/[sandbox]/sandbox.ts [hunk 3]
```diff
@@ -140,8 +139,8 @@ export class Sandbox extends LitElement {
               person-id="30313-Carl"
               show-filters
               base-url="https://cjaas-devus2.azurewebsites.net"
-              .stream-read-token=${getSToken()}
-              .tape-read-token=${getTToken()}
+              tape-read-token="so=demoassure&sn=sandbox&ss=tape&sp=r&se=2022-11-23T20:33:44.019Z&sk=journeyUi&sig=Msa4zTsNmkeDHJcmQuXUVHTTzs1KATCQ%2FDNrVR2O7eU%3D"
+              stream-read-token="so=demoassure&sn=sandbox&ss=stream&sp=r&se=2022-11-23T20:30:20.765Z&sk=journeyUi&sig=76cI1nBPkA0HdQved8YHiTQbOThPOR8W5UdwZzeUuPc%3D"
```
Comment (elenagiere) [inline Timeline/src/[sandbox]/sandbox.ts:143]: why did you remove the dot notation in front of each property?

File: Timeline/src/index.ts [hunk 4]
```diff
@@ -174,27 +194,28 @@ export default class CjaasTimelineWidget extends LitElement {
...
       this.eventSource = new EventSource(
         `${this.baseURL}/v1/journey/streams/${this.personId}?${this.streamToken}`,
-        header
+        header,
       );
     }
 
     if (this.eventSource) {
-      this.eventSource!.onmessage = (event: ServerSentEvent) => {
+      this.eventSource.onmessage = (event: ServerSentEvent) => {
```
Comment (elenagiere) [inline Timeline/src/index.ts:207]: why remove the exclamation mark, doesn't that help prevent calling on an undefined object?

File: Timeline/src/index.ts [hunk 5]
```diff
@@ -174,27 +194,28 @@ export default class CjaasTimelineWidget extends LitElement {
...
           data = JSON.parse(event.data);
+          data.time = DateTime.fromISO(data.time);
           this.newestEvents = [data, ...this.newestEvents];
         } catch (err) {
           console.log("Event Source Ping");
         }
       };
 
-      this.eventSource!.onerror = () => {
+      this.eventSource.onerror = () => {
```
Comment (elenagiere) [inline Timeline/src/index.ts:218]: why remove the exclamation mark, doesn't that help prevent calling on an undefined object?

File: Timeline/package.json (hunk 2 above)
Comment (venkateshwarv) [reply Timeline/package.json:19]: I had it by passed in my machine. This change got leaked. I will add this back.

File: Timeline/src/[sandbox]/sandbox.ts (hunk 3 above)
Comment (venkateshwarv) [reply Timeline/src/[sandbox]/sandbox.ts:143]: with dot notation it should have been `.tapeReadToken` We dint need it with static text.

The SAS tokens generated in browser was not valid for some reason.

File: Timeline/src/index.ts (hunk 5 above)
Comment (venkateshwarv) [reply Timeline/src/index.ts:218]: we check for this in line 186 `    if (this.eventSource) {` already

File: CustomerJourney/package.json (hunk 1 above)
Comment (venkateshwarv) [reply CustomerJourney/package.json:29]: Sure.

Comment (venkateshwarv) [reply CustomerJourney/package.json:29]: Sure

File: Timeline/src/index.ts (hunk 5 above)
Comment (elenagiere) [reply Timeline/src/index.ts:218]: ok

**Review by elenagiere — APPROVED** [review]

**Pull request merged**